### ISSUE TITLE
TypeError: e.ref is not a function

### ISSUE THREAD

**Issue by IsaiahJTurner** [firebase/emberfire#339]
Was there a change in how getting the ref works?

**Comment by tstirrat** [comment]
Please give more context. Where are you seeing this?

`model.ref()` is implemented in the ember initializer by monkey patching the DS.Model class prototype.

Are your initializers running?

**Comment by tstirrat** [comment]
You can see the code over the last few releases. I did not change anything with `model.ref()`

**Comment by IsaiahJTurner** [comment]
Ah, I see what happened.

It was from me trying to fix this deprecation
> DEPRECATION: The `initialize` method for Application initializer 'emberfire' should take only one argument - `App`, an instance of an `Application`. [deprecation id: ember-application.app-initializer-initialize-arguments] See http://emberjs.com/deprecations/v2.x/#toc_initializer-arity for more details.

I removed the param specified below.
```js

import Ember from 'ember';
import Resolver from 'ember/resolver';
import loadInitializers from 'ember/load-initializers';
import config from './config/environment';

var App;

Ember.MODEL_FACTORY_INJECTIONS = true;

App = Ember.Application.extend({
  modulePrefix: config.modulePrefix,
  podModulePrefix: config.podModulePrefix,
  Resolver: Resolver
});

loadInitializers(App, config.modulePrefix); // removed "config.modulePrefix"

export default App;

```

The rest of the app and modules seem to work fine. I'm still trying to figure out if this something to do with Firebase or my own issue.

**Comment by tstirrat** [comment]
hrm... #328 should have removed the deprecation warning.. are you on the latest version of emberfire?

**Comment by tstirrat** [comment]
should be fixed in 1.6.1
